generators/server/templates/src/main/resources/config/_bootstrap.yml

```diff
@@ -0,0 +1,10 @@
+# ===================================================================
+# Spring Cloud Config bootstrap configuration
+# ===================================================================
+
+spring:
+    application:
+        name: <%= baseName %>
+    cloud:
+        config:
+            uri: http://localhost:8761/config
```

Comment:
Good question! We already have that issue with the registry: if no registry is running, the application runs fine but we get ugly stacktraces